src/Libm.jl

```diff
@@ -1,5 +1,8 @@
 module Libm
+using Base.Math.@horner
```

Comment:
you're not extending it, so actually `using` is better if you're asking for a specific function